dipper/sources/OMIMSource.py

```diff
@@ -0,0 +1,223 @@
+
+        myfile = '/'.join((self.rawdir, self.mimfiles[src_key]['file']))
+
+        col = self.mimfiles[src_key]['columns']
+        with open(myfile, 'r') as readfile:
+            reader = csv.reader(readfile, delimiter='\t')
+            row = next(reader)  # copyright
+            row = next(reader)  # date  generated
+            row = next(reader)  # column header
+            row[0] = row[0][2:]  # remove octothorp '# '
+            if not self.check_fileheader(col, row):
+                exit(-1)
+
+            for row in reader:
+                if row[0][0] == '#':  # they have comments at the end
+                    continue
+
+                declared = row[col.index('Prefix')].strip()
+                omim_id = row[col.index('Mim Number')].strip()
+                pref_label = row[col.index('Preferred Title; symbol')].strip()
+                # alt_label =row[col.index('Alternative Title(s); symbol(s)')].strip()
+                # inc_label = row[col.index('Included Title(s); symbols')].strip()
+
+                if declared == 'Caret':  # moved|removed|split -> moved twice
+                    # populating a dict from an omim to a set of omims
+                    self.omim_type[omim_id] = self.globaltt['obsolete']
+                    self.omim_replaced[omim_id] = []
+                    if pref_label[:9] == 'MOVED TO ':
+                        token = pref_label.split(' ')
+                        rep = token[2]
+                        if not re.match(r'^[0-9]{6}$', rep):
+                            LOG.error('Report malformed omim replacement %s', rep)
+                            # clean up ones I know about
+                            if rep[0] == '{' and rep[7] == '}':
+                                rep = rep[1:7]
+                                LOG.info('Repaired malformed omim replacement %s', rep)
+                            if len(rep) == 7 and rep[6] == ',':
+                                rep = rep[:6]
+                                LOG.info('Repaired malformed omim replacement %s', rep)
+                        if len(token) > 3:
+                            self.omim_replaced[omim_id] = [rep, token[4]]
+                        else:
+                            self.omim_replaced[omim_id] = [rep]
+
+                elif declared == 'Asterisk':
+                    self.omim_type[omim_id] = self.globaltt['gene']
+                elif declared == 'NULL':
+                    self.omim_type[omim_id] = self.globaltt['Suspected']   # NCIT:C71458
+                elif declared == 'Number Sign':
+                    self.omim_type[omim_id] = self.globaltt['Phenotype']
+                elif declared == 'Percent':
+                    self.omim_type[omim_id] = self.globaltt[
+                        'heritable_phenotypic_marker']
+                elif declared == 'Plus':
+                    self.omim_type[omim_id] = self.globaltt['has_affected_feature']
```

Comment:
My understanding is that downstream this is treated like a gene, or are there cases where it is treated like other types?  I think typing a class/individual as a object property could be potentially confusing even if it doesn't make it's way to the RDF model.

Reply:
which `this` are we referring to?

only omim identifiers of type 'gene' are being declared subclasses of the type
 as to what types to consider each of the choices I'm 100% open
but they each need a different type here.

Reply:
Referring to:

    elif declared == 'Plus':
        self.omim_type[omim_id] = self.globaltt['has_affected_feature']

We've previously considered these genes, for example:
https://monarchinitiative.org/gene/OMIM:100650

Reply:
I think we are ok,  these types are not binding in rdf-land
just a way to make as fine a distinction as we can. 
https://github.com/TomConlin/dipper/blob/master/dipper/sources/OMIM.py#L392